File: JAVA PROGRAMS/trash.java
public class trash {
    public static void main(String[] args) {
        int num = 123454321;
        int num1 = num;
        int rev = 0;
        while(num>0){
            int d = num%10;
            rev = (rev*10)+d;
            num/=10;
        }
        if(num1 == rev)System.out.println("Pallindrome");
        else System.out.println("Not pallindrome");
    }
}

File: C SHARP PROGRAMS/CSharpProjects/Programs/trash.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CSharpProjects
{
    internal class trash
    {
        static void Main(string[] args)
        {
            string str = "samabs";
            int low = 0;
            int high = str.Length - 1;

            bool check = true;
            while(low<= high)
            {
                if (str[low] != str[high])
                {
                    check = false;
                    break;
                }
                low++;high--;
            }
            if(check == true)
            {
                Console.WriteLine("Pallindrome");

            }
            else
            {
                Console.WriteLine("Not");
            }
            
        }
    }
}
